packages/frontend/app/util/getCollectionIndexForDate.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { afterAll, beforeAll, expect, test, vitest } from "vitest";
+import type { CreateChannelDTO } from "@marcianosrs/engine";
+import { getPollIndexForDate } from "./getCollectionIndexForDate";
+
+export const mockChannel: CreateChannelDTO = {
+	name: "Test Channel",
+	theme: {
+		type: "default",
+		data: {},
+	},
+	createdBy: "user-id",
+	frequency: {
+		cronExpression: "0 10 * * 1-5",
+		description: "Every weekday around 10 'o' clock",
+	},
+	collection: [
```

Comment:
we hebben all los 'poll' document toch, dat document Id is denk ik wat je hier wil ipv plugin Id en plugin Type?
betekend dit dat bij elk antwoord wat moet gegeven worden de channel geupdate moet worden in firebase? Is dat bij huidige poll app ook zo? werkte dat goed?

Reply:
Ja dat klopt!